autoblocks/_impl/testing/run.py

```diff
@@ -300,7 +250,16 @@ def run_test_suite(
     # How many evaluators to run concurrently on the result of a test case
     max_evaluator_concurrency: int = 5,
 ):
-    init_global_vars_and_event_loop()
+    cli_server_address = AutoblocksEnvVar.CLI_SERVER_ADDRESS.get()
+    if not cli_server_address:
+        raise RuntimeError(
+            "Autoblocks tests must be run within the context of the testing CLI.\n"
+            "Make sure you are running your test command with:\n"
+            "$ npx autoblocks testing exec -- <your test command>"
+        )
+
+    if not event_loop_started():
```

Comment:
we should just call `global_state.init()`. init takes care of checking whether or not it's already started